qutil.c

```diff
@@ -215,6 +215,21 @@ load_file_nostat(int fd, size_t *total)
 	return (buf);
 }
 
+int
+ipv6_supported(void)
+{
+	FILE *fp;
+	int enabled = 0;
+
+	fp = fopen("/proc/sys/net/ipv6", "r");
+	if (fp != NULL) {
+		enabled = 1;
+		fclose(fp);
+	}
+
+	return enabled;
+}
+
```

Comment:
I'm ok with this but a part of me thinks we should be trying to create an AF_INET6 socket instead of opening /proc.
like
```
fd = socket(AF_INET6, SOCK_STREAM, 0)
enable = fd != -1
```